docker/tensorflow/gpu/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM nvidia/cuda:11.0.3-cudnn8-runtime-ubuntu20.04
+
+RUN apt-get update \
+        && apt-get install -y --no-install-recommends \
+            apt-transport-https \
+            git \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
+        && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+            build-essential \
+            libgl1-mesa-glx \
+            libglib2.0-dev \
+            wget \
+            curl \
+            zip \
+            unzip \
+            nano \
+            openssh-server \
+            openssh-client \
+            sudo \
+            python3 \
+            python3-dev \
+            python3-pip \
+        && cd /usr/bin \
+        && ln -s python3.8 python \
+        && rm -rf /var/lib/apt/lists/*
+
+RUN pip3 install --upgrade pip \
+        && pip3 install setuptools
+
+ENV PATH /usr/local/nvidia/bin:/usr/local/cuda/bin:${PATH}
+ENV LD_LIBRARY_PATH /usr/local/nvidia/lib:/usr/local/nvidia/lib64
+
+# nvidia-container-runtime
+ENV NVIDIA_VISIBLE_DEVICES all
+ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
+
+ARG https_proxy
```

Comment:
http_proxy missed.

P.S. Use ENV for proxy is bad practice, you should rebuild image to change proxy setting.

Reply:
Thanks for your comment! I fixed it.